scripts/tests/chiptest/__init__.py

```diff
@@ -160,6 +160,57 @@ def _GetInDevelopmentTests() -> Set[str]:
     }
 
 
+def _GetDarwinFrameworkToolUnsupportedTests() -> Set[str]:
+    """Tests that fail in darwin-framework-tool for some reason"""
+    return {
+        "DL_LockUnlock",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_AddNewFabricFromExistingFabric",  # darwin-framework-tool does not support the GetCommissionerRootCertificate command.
+        "TestAttributesById",  # The name of the arguments once converted differs for chip-tool and darwin-framework-tool
+        "TestBasicInformation",  # darwin-framework-tool does not support writing readonly attributes by name
+        "TestClusterComplexTypes",  # Test Failure : The test expectation "wasPresent (0) == True" is false
+        "TestClusterMultiFabric",  # When reading TestFabricScoped, the result differs
```

Comment:
Why does it differ?

Reply:
I have tried to add a comment to explain it.